augur/filter/include_exclude_rules.py

```diff
@@ -1,9 +1,9 @@
-import ast
 import json
 import operator
 import re
 import numpy as np
 import pandas as pd
+from pandas.core.computation.parsing import tokenize_string, BACKTICK_QUOTED_STRING
```

Comment:
I don't love using Pandas private internals here when we could go with https://github.com/nextstrain/augur/pull/1417 instead. But will leave it to your judgement!

Reply:
I think using Pandas private internals is ok, since it addresses the query parsing with the same functions that Pandas uses later on when the query is applied.

My concern with the current approach here is the inability to distinguish variable names from other names. If there was a function `pandas.core.computation.parsing.extract_variables`, I would be in favor of using that. Since that doesn't exist, and this implementation is half-baked, I'll continue with #1417.